.github/workflows/backend.yml

```diff
@@ -0,0 +1,31 @@
+name: backend-CI
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
It isn't possible to push onto the main branch, is it?

Reply:
I belive that fires on a PR merge to master. Useful for checking that the rebase didn't do anything funky. (altough we've got the up to date rule so this is *probably* not needed)